database/knexfile.js

```diff
@@ -9,7 +9,14 @@ module.exports = {
     schemaName: env.PGSCHEMA,
   },
   pool: {
-    min: process.env.KNEX_POOL_MIN || 1,
-    max: process.env.KNEX_POOL_MAX || 100,
+    min: +process.env.KNEX_POOL_MIN || 0,
```

Comment:
@k1ch  what are the `+` signs on these two lines?

Reply:
@mikelax it converts string to number. Those Knex configs should be number otherwise it throws an error.